packages/components/src/stories/SquiggleChart/Calculator.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { SquiggleChart } from "../../components/SquiggleChart.js";
+
+const meta = {
+  component: SquiggleChart,
+} satisfies Meta<typeof SquiggleChart>;
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Basic: Story = {
+  args: {
+    code: `
+    Calculator.make(
+      {
+        fn: {|a,b,c|a+b+c},
+        rows: ["a", "b", "c"]
+      }
+    )
```

Comment:
Is the custom Calculator object necessary at this stage?

We could render functions as calculators by default (which seems useful anyway).

I'm pretty sure you could extract parameter names through introspection.

Though there's a corner case of (number => number) and (number => dist) to consider.

Reply:
That's a good question. I don't know if "necessary" is the right word, but it seems like a good thing to have. Note that with custom calculators, it's possible to give default values, proper string names, descriptions, etc. 

Maybe we could do auto-calculators later. Much of the work is similar, at least.